src/main/resources/templates/index.html

```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE html>
```

Comment:
This page should contain spaces for:

>  Informationen über das Hotel wie z.B. Kontakt, Anreise, Aktionen, Wissenswertes, ...

Those spaces can be filled with Lorem Ipsum for now

Reply:
done (kurzes lorem..)